mediachain/indexer/mc_simpleclient.py

```diff
@@ -294,6 +294,7 @@ def get_entities(self, *args, **kw):
     
         
     def get_objects(self,
+                    catchup_blockchain = True, # TODO, replace with last_block_height, once client support is in
```

Comment:
hmm we could probably use a `block_depth = 0` here instead, but for providing continuous/resumable tracking we actually do want height. Hmm, I guess let's just leave this for now